packages/account/src/Configs/personal-details-config.ts

```diff
@@ -167,7 +165,7 @@ export const personal_details_config = ({
             rules: [['req', localize('Employment status is required.')]],
         },
         tax_identification_confirm: {
-            default_value: false,
+            default_value: '',
```

Comment:
this value will be used for a checkbox as a boolean, we don't need to convert it to a string